keras/engine/training_utils.py

```diff
@@ -448,9 +458,9 @@ def standardize_weights(y,
         ValueError: In case of invalid user-provided arguments.
     """
     if sample_weight_mode is not None:
-        if sample_weight_mode != 'temporal':
+        if sample_weight_mode != 'temporal' and sample_weight_mode != 'element':
```

Comment:
`if sample_weight_mode not in ('temporal', 'element'):`

Reply:
I wrote a [test script](https://github.com/keras-team/keras/issues/10561#issuecomment-410533376), but I'd need help to turn it into test units. (As I mention in the [Keras API Design Review google doc](https://docs.google.com/document/d/19BDXgNmeTgpgb9xYKzNboXyM7XX2PeM3mlvCFCdIQj0/edit?usp=sharing), I've tried following the [instructions to test keras](https://keras.io/contributing/#pull-requests), but it fails even with the unpatched main keras branch.)

I've pushed your suggested change in commit 86b241c7dfd63070d8864f88002420996576fdb8.